cli/enginecli/test_lootbox.py

```diff
@@ -172,6 +203,33 @@ def _open_lootbox(self, account, lootboxId, count=None):
                     token_amount * count,
                 )
 
+            elif reward_type == 1:
```

Comment:
You should check the `total_supply` on the pool before and after minting, as well.

Reply:
Done